clusterloader2/testing/load/modules/measurements.yaml

```diff
@@ -16,6 +16,9 @@
 {{$ENABLE_SYSTEM_POD_METRICS:= DefaultParam .ENABLE_SYSTEM_POD_METRICS true}}
 {{$ENABLE_VIOLATIONS_FOR_API_CALL_PROMETHEUS := DefaultParam .CL2_ENABLE_VIOLATIONS_FOR_API_CALL_PROMETHEUS false}}
 {{$ENABLE_VIOLATIONS_FOR_API_CALL_PROMETHEUS_SIMPLE := DefaultParam .CL2_ENABLE_VIOLATIONS_FOR_API_CALL_PROMETHEUS_SIMPLE true}}
+{{$ENABLE_CEP_PROPAGATION_DELAY_MEASUREMENT := DefaultParam .CL2_ENABLE_CEP_PROPAGATION_DELAY_MEASUREMENT false}}
+{{$CEP_PROPAGATION_DELAY_SLO_BUCKET := DefaultParam .CL2_CEP_PROPAGATION_DELAY_SLO_BUCKET ""}}
```

Comment:
Nitpick: shall we add the default SLO bucket here as well, like we do for the SLO percentile line below?

Reply:
Oh, I didn't realize that... Would it be possible then to treat this parameter as a floating-point number (i.e. `{{$CEP_PROPAGATION_DELAY_SLO_BUCKET := DefaultParam .CL2_CEP_PROPAGATION_DELAY_SLO_BUCKET 600}}`) instead and parse it to string inside the measurement's logic using `FormatFloat` method like [here](https://go.dev/play/p/HBXtSUSDCy2)?

Reply:
Yes, thank you for the suggestion.
Done. Also changed it in the unit test for custom SLOs to have float value for this parameter.